contrib/signet/README.md

```diff
@@ -0,0 +1,19 @@
+Contents
+========
+This directory contains tools related to Signet, both for running a Signet yourself and for using one.
+
+getcoins.py
+===========
+
+A script to call a faucet to get Signet coins.
+
+Syntax: `getcoins.py [-h|--help] [-c|--cmd=<bitcoin-cli path>] [-f|--faucet=<faucet URL>] [-a|--addr=<signet bech32 address>] [-p|--password=<faucet password>] [--] [<bitcoin-cli args>]`
+
+* `--cmd` lets you customize the bitcoin-cli path. By default it will look for it in the PATH
```

Comment:
nit: Instead of duplicating the help with varying verbosity and details, you could write it one (in python) and then just mention in the readme to call `getcoins.py --help` to get the up-to-date help.

This avoid conflicting and outdated markdown help texts.